lib/cartopy/feature.py

```diff
@@ -162,7 +162,8 @@ def __init__(self, category, name, scale, **kwargs):
         * scale:
             The dataset scale, i.e. one of '10m', '50m', or '110m'.
             Corresponding to 1:10,000,000, 1:50,000,000, and 1:110,000,000
-            respectively.
+            respectively. If set to 'auto', the scale will be automatically
```

Comment:
But this isn't true for all Natural Earth features... I thought it was just those with ``10m``, ``50m`` and ``110m`` scales?